gui-comp-int-calculator.py

```diff
@@ -107,6 +107,10 @@ def OnButtonClick(self):
         intearn = round((amount - p),2)
         self.amt.set(amount)
         self.earn.set(intearn)
```

Comment:
It's been a while since I took a look at this project, but I think the lines
```
self.amt.set(amount)
self.earn.set(intearn)
```

should be deleted, given your changes below.